examples/2-scaffolding/src/models/PokemonAttackModel.ts

```diff
@@ -2,24 +2,20 @@ import { Instance } from "mobx-state-tree"
 import { PokemonAttackModelBase } from "./PokemonAttackModel.base"
 
 /* The TypeScript type of an instance of PokemonAttackModel */
-export interface PokemonAttackModelType
-  extends Instance<typeof PokemonAttackModel.Type> {}
+export interface PokemonAttackModelType extends Instance<typeof PokemonAttackModel.Type> {}
```

Comment:
@zenflow some of these formatting changes are reverted by prettier for me, do you know if prettier is running pre-commit for you as it should?

Reply:
@chrisdrackett No, it doesn't seem to be running. 

These formatting changes are the result of running `yarn test` which involves `cd`ing into `examples/2-scaffolding` and running the `scaffold` script which uses "force" option, so it looks like prettier is fighting with our generated code here.

But that should be ok if the commit hook works; I'll sort that out.

I'll make a commit now to format everything with prettier.

Reply:
cool! I wonder if something around husky isn't working on windows?

Reply:
lol seems safe to assume that being on windows is the problem 💩